### PR TITLE
Login UI

### PR DESCRIPTION
**Pull request by HenriLouw**
I accidentally created the feed on the login branch... So you guys can get a 2 for 1 special when reviewing XD. I'm not done with the feed yet; I just want to get it out so long that I don't block Milahn. I will probably finish the rest tonight. As requested yesterday, I'll fix my commit names going forward...

### CODE REVIEW

**Review by milahnmartin — CHANGES_REQUESTED** [review]
Everything looks good, just the inconsistency with the import annotations please. :)

File: src/views/Map.jsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import LoadingIndicator from '@/components/ui/LoadingIndicator';
 import { auth } from '@/utils/firebase-config';
 import UserInformation from '../components/ui/UserInformation';
 import Sidebar from '../components/Sidebar';
```
[the remark this answers is not in the document]
Comment (HenriLouw) [reply src/views/Map.jsx]: I did not create this... But I will fix it for you :)

File: src/components/ui/button.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { cn } from '../utils';
```
[the remark this answers is not in the document]
Comment (HenriLouw) [reply src/components/ui/button.jsx:1]: I did not create this... But I will fix it for you :)

File: src/views/Map.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (milahnmartin) [reply src/views/Map.jsx]: my bad, no idea why I did that, the people who reviewed didnt pick it up I guess :()

**Comment by Louis-Bat17** [comment]
After el reviews, tis looking lit, thanks Henri!

**Review by milahnmartin — APPROVED** [review]
Looks good thank you !

**Review by Louis-Bat17 — APPROVED** [review]

**Pull request merged**